LoopKitUI/Views/ConfigurationPage.swift

```diff
@@ -87,7 +87,7 @@ extension ConfigurationPage {
     ) {
         self.init(
             title: title,
-            actionButtonTitle: Text("Save", comment: "The button text for saving on a configuration page"),
+            actionButtonTitle: Text(LocalizedString("Save", comment: "The button text for saving on a configuration page")),
```

Comment:
Why is this correct and the previous form not correct?

Reply:
The previous form will look up translations in the app's bundle. The LocalizedString method is a helper that looks in the local framework's bundle for translations.